apps/pact-broker/pact-broker/sbox-intsvc.yaml

```diff
@@ -4,8 +4,10 @@ metadata:
   name: pact-broker
   namespace: pact-broker
 spec:
+  chart:
+    spec:
+      version: 0.4.0
```

Comment:
Requires https://github.com/hmcts/chart-pact-broker/pull/33 to be released before merge of this